src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.scss

```diff
@@ -76,3 +75,11 @@ $double_standard_gutter: calc($standard-gutter * 2);
   column-gap: 40px;
   width: 268px;
 }
+
+.showHideWrapper {
+  --show-hide-chevron-height: var(--chevron-height, 8px);
+}
+
+.rulerContainerClass {
```

Comment:
No need for the `Class` suffix?

Reply:
I agreed no need for class but when i removed the Class the name `rulerContainer` clashes with the ref variable name `rulerContainer` so i have renamed it to `rulerWrapper`

Reply:
Sorry, I meant that the word "Class" is unnecessary in a CSS file, because anything starting with a `.` is a class name.

In tsx, when you are writing:

![image](https://user-images.githubusercontent.com/6834224/185094669-3a621589-12ac-408f-9eed-ac4020922772.png)

— the word `Class`, or, better, `ClassName`, is actually very helpful, to the point of being necessary:

```ts
// rulerWrapperClassName, or rulerWrapperClass, or rulerWrapperClasses, or rulerWrapperClassNames
const rulerWrapperClassName = classNames(
  styles.resultsSummaryRow,
  styles.rulerWrapper // here, though, it would still be styles.rulerWrapper; not styles.rulerWrapperClass
);
```